src/Common/Entity/Row.php

```diff
@@ -38,12 +38,13 @@ public function __construct(array $cells, ?Style $style = null)
 
     /**
      * @param list<null|bool|DateInterval|DateTimeInterface|float|int|string> $cellValues
+     * @param Style[]                                                         $columnStyles
```

Comment:
Currently this cannot be merged *as-is*: the `$cellValues` argument is typed as `list<>`, and relying on `$columnStyles` to have the same keys can't be requested nor enforced.

But a new API is welcome, so please:

1. Add a new `fromValuesWithStyle` API with both `$cellValues` and `$columnStyles` typehinted as `array<array-key, [...]>`
2. Add tests for the new API
3. Add brief mention to the docs for the new API

Reply:
Thanks for the suggestion! I've made the required changes in this commit: [ac52867](https://github.com/openspout/openspout/pull/235/commits/ac5286704294654d7e2ae042ea587cb2b129e5f5)